src/lib/dnssd/minimal_mdns/Server.cpp

```diff
@@ -121,51 +121,38 @@ class InterfaceTypeFilterDelegate : public ServerBase::BroadcastSendDelegate
 namespace BroadcastIpAddresses {
 
 // Get standard mDNS Broadcast addresses
-
-void GetIpv6Into(chip::Inet::IPAddress & dest)
+chip::Inet::IPAddress Get(chip::Inet::IPAddressType addressType)
 {
-    if (!chip::Inet::IPAddress::FromString("FF02::FB", dest))
+    chip::Inet::IPAddress address;
+#if INET_CONFIG_ENABLE_IPV4
+    if (addressType == chip::Inet::IPAddressType::kIPv4)
     {
-        ChipLogError(Discovery, "Failed to parse standard IPv6 broadcast address");
-    }
-}
+        if (!chip::Inet::IPAddress::FromString("224.0.0.251", address))
+        {
+            ChipLogError(Discovery, "Failed to parse standard IPv4 broadcast address");
 
-void GetIpv4Into(chip::Inet::IPAddress & dest)
-{
-    if (!chip::Inet::IPAddress::FromString("224.0.0.251", dest))
+            // not valid, however the parsing should never fail
+            return chip::Inet::IPAddress::Any;
+        }
+    }
+    else
+#endif
     {
-        ChipLogError(Discovery, "Failed to parse standard IPv4 broadcast address");
+        if (!chip::Inet::IPAddress::FromString("FF02::FB", address))
+        {
+            ChipLogError(Discovery, "Failed to parse standard IPv6 broadcast address");
```

Comment:
Please just VerifyOrDie. This will never happen at runtime and you are dead if it happens: other stuff will fail.